docs/components/ActionLabel/Mobile.stories.mdx

```diff
@@ -1,74 +0,0 @@
-import { ArgsTable, Canvas, Meta, Story } from "@storybook/addon-docs";
-import { View } from "react-native";
-import { Figma } from "storybook-addon-designs/blocks";
-import { ActionLabel } from "@jobber/components-native";
-
-<Meta title="Components/Actions/ActionLabel/Mobile" component={ActionLabel} />
-
-## Action Label
-
-```ts
-import { ActionLabel } from "@jobber/components-native";
-```
-
-<Canvas>
-  <Story name="ActionLabel" args={{ children: "I am a label text" }}>
-    {args => <ActionLabel {...args}>{args.children}</ActionLabel>}
-  </Story>
-</Canvas>
-
-<ArgsTable of={ActionLabel} />
-
-### Font color variation
-
-<Canvas>
-  <Story
-    name="Font color variation"
-    parameters={{
-      backgrounds: {
-        default: "light",
-      },
-    }}
-    args={{ variation: "learning", children: "Learning" }}
-  >
-    {args => <ActionLabel {...args}>{args.children}</ActionLabel>}
-  </Story>
-</Canvas>
-
-### Disabled
-
-As a best practice, do not design with disabled button states. This has negative
-impacts on accessibility as well as an increase in complexity for users to
-understand why the interface is disabled and how to resolve it.
-
-With that said, if you can't design a flow without a disabled state, this is how
-you disable an `ActionLabel`.
```

Comment:
Maybe we should keep this comment about disabled as it's related to accessibility, or add this comment to the usage guidelines. What do you think?

Reply:
hmm good point!